test-specifications/crypto-service/accountDeleteTransaction.md

```diff
@@ -90,12 +94,11 @@ The tests contained in this specification will assume that a valid account was a
 
 - The ID of the account to which to transfer remaining balances.
```

Comment:
Also need to add happy path.

Reply:
I'm not sure I understand what you mean

Reply:
I mean positive case, like "Deletes an account with a transfer account that is valid"

Reply:
Ah I see, I thought about that but figured it would be a redundant test of no. 1 for `deleteAccountId`

Reply:
Agree with you. 
Maybe, just for some clarity, we could change it in test of no. 1 for `deleteAccountId` -- do without transfer account, and then in transfer account cases add "Deletes an account with a transfer account that is valid", if it will make sense for you.

Reply:
Sure, fixed